README.md

```diff
@@ -34,64 +38,34 @@ yarn test_sband <options>
 
 ## Using This Repository with a UHF SDR
 
-If using this repository for FlatSat testing or deployment, follow these slightly different instructions. Hardware setup steps are excluded.
-
-Step 1: Check if you have conda installed by running `conda`. If not installed, follow the [conda installation instructions](https://docs.conda.io/projects/conda/en/latest/user-guide/install/index.html#installing-conda-on-a-system-that-has-other-python-installations-or-packages).
+If using this repository with the satellites or the flatsat, follow these instructions. Hardware setup steps are excluded.
 
-Step 2: Set up GNURadio
+Step 4: Install conda (if not already present) by following the [conda installation instructions](https://docs.conda.io/projects/conda/en/latest/user-guide/install/index.html#installing-conda-on-a-system-that-has-other-python-installations-or-packages). We use a conda environment to run GNURadio, because there are additional GNURadio dependencies (out-of-tree modules) that we use in the ground station.
```

Comment:
y this start at step 4

Reply:
To indicate that the previous 3 steps still need to be followed